src/admin.rs

```diff
@@ -671,6 +677,34 @@ where
     }
 }
 
+/// Send response packets for shutdown.
+async fn shutdown<T>(stream: &mut T) -> Result<(), Error>
+where
+    T: tokio::io::AsyncWrite + std::marker::Unpin,
+{
+    let mut res = BytesMut::new();
+
+    res.put(row_description(&vec![("success", DataType::Text)]));
+
+    let mut shutdown_success = "T";
+
+    let pid = std::process::id();
+    if signal::kill(Pid::from_raw(pid.try_into().unwrap()), Signal::SIGINT).is_err() {
+        error!("Unable to send SIGINT to PID: {}", pid);
+        shutdown_success = "F";
+    }
+
+    res.put(data_row(&vec![shutdown_success.to_string()]));
+
+    res.put(command_complete("SHUTDOWN"));
```

Comment:
Typically this would return a `SELECT 1`, otherwise clients could get confused? Not entirely sure. If you're returning rows, you want to double check that you're using the right protocol syntax.

Reply:
Currently all of our other custom protocol commands which return rows follow this pattern, I can change this to a select 1 but it would be different from the rest of our commands

Reply:
You're absolutely right. Disregard my comment.